lib/led/ledcontrol.js

```diff
@@ -704,6 +711,128 @@ Controllers = {
     },
   },
 
+  HT1632C: {
+    OP: {
+      SYS_DIS: 0x00,
+      SYS_EN: 0x01,
+      LED_OFF: 0x02,
+      LED_ON: 0x03,
+      BLINK_OFF: 0x08,
+      BLINK_ON: 0x09,
+      SLAVE_MODE: 0x10,
+      MASTER_MODE: 0x14,
+      INT_RC: 0x18,
+      EXT_CLK: 0x1C,
+      PWM_CONTROL: 0xA0,
+
+      // Should think about naming these better or whatever.
+      COMMON_8NMOS: 0x20,
+      COMMON_16NMOS: 0x24,
+      COMMON_8PMOS: 0x28,
+      COMMON_16PMOS: 0x2C
+    },
+
+    initialize: function(opts) {
+      this.pins = {
+        data: opts.pins.data,
+        write: opts.pins.write,
+        cs: opts.pins.cs
+      };
+      ["data", "write", "cs"].forEach(function(pin) {
+        this.io.pinMode(this.pins[pin], this.io.MODES.OUTPUT);
+      }, this);
+
+      this.ledMatrix = [];
+
+      this.sendCommand(this.OP.SYS_EN);
+      this.sendCommand(this.OP.LED_ON);
+      this.sendCommand(this.OP.BLINK_OFF);
+      this.sendCommand(this.OP.MASTER_MODE);
+      this.sendCommand(this.OP.INT_RC);
+      // hardcoding this for now. need to figure out a nice way of configuring this.
+      this.sendCommand(this.OP.COMMON_16NMOS);
+      this.sendCommand(this.OP.PWM_CONTROL | 0xF);
+    },
+
+    clearScreen: function() {
+      var width = 24, height = 16;
+      for (i=0; i<=(width*height/8); i++) {
+        this.ledMatrix[i] = Math.floor( Math.random() * 256 );
+      }
+      this.writeScreen();
+    },
+
+    /**
+     * Write the data in the buffers to the screen.
+     * @return {[type]} [description]
+     */
+    writeScreen: function() {
```

Comment:
This looks like `writeDisplay`. https://github.com/ericandrewlewis/johnny-five/blob/issue-1000/lib/led/ledcontrol.js#L622-L630